include/vast/Dialect/LowLevel/LowLevelOps.td

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2021-present, Trail of Bits, Inc.
+
+#ifndef VAST_DIALECT_IR_LOWLEVELOPS
+#define VAST_DIALECT_IR_LOWLEVELOPS
+
+include "mlir/Interfaces/CallInterfaces.td"
+include "mlir/IR/BuiltinAttributes.td"
+include "vast/Interfaces/SymbolInterface.td"
+
+def LowLevel_StructGEPOp
+  : LowLevel_Op< "gep" >
+  , Arguments<(ins AnyType:$record, I32Attr:$idx, StrAttr:$name)>
```

Comment:
This should be some PointerLikeType or ObjectLikeType, at least add TODO to fix later.
Same for result type?